init.lua

```diff
@@ -371,31 +372,34 @@ end
 local function lootItem(index, doWhat, button, qKeep)
     loot.logger.Debug('Enter lootItem')
     if not shouldLootActions[doWhat] then return end
-    local corpseItemID = mq.TLO.Corpse.Item(index).ID()
-    local itemName = mq.TLO.Corpse.Item(index).Name()
+    local corpseItem = mq.TLO.Corpse.Item(index)
+    local corpseItemID =corpseItem.ID()
+    local itemName = corpseItem.Name()
+    local itemLink = corpseItem.ItemLink('CLICKABLE')()
     mq.cmdf('/nomodkey /shift /itemnotify loot%s %s', index, button)
     -- Looting of no drop items is currently disabled with no flag to enable anyways
-    -- added check to make sure the cursor isn't empty so we can exit the pause early.-- or not mq.TLO.Corpse.Item(index).NoDrop() 
-    mq.delay(1)
-    if doWhat == 'Destroy' and mq.TLO.Cursor.ID() == corpseItemID then mq.cmd('/destroy') end
+    -- added check to make sure the cursor isn't empty so we can exit the pause early.-- or not mq.TLO.Corpse.Item(index).NoDrop()    
     mq.delay(1) -- for good measure.
     mq.delay(5000, function() return mq.TLO.Window('ConfirmationDialogBox').Open() or mq.TLO.Cursor() == nil end)
     if mq.TLO.Window('ConfirmationDialogBox').Open() then mq.cmd('/nomodkey /notify ConfirmationDialogBox Yes_Button leftmouseup') end
     mq.delay(5000, function() return mq.TLO.Cursor() ~= nil or not mq.TLO.Window('LootWnd').Open() end)
     mq.delay(1) -- force next frame
     -- The loot window closes if attempting to loot a lore item you already have, but lore should have already been checked for
     if not mq.TLO.Window('LootWnd').Open() then return end
-    report('%sing \ay%s\ax', doWhat, itemName)
+    if doWhat == 'Destroy' then mq.cmd('/destroy') end
```

Comment:
i'd keep `and mq.TLO.Cursor.ID() == corpseItemID` in the if, just for safety

Reply:
added.